apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -370,6 +371,17 @@ const Ticket = new GQLListSchema('Ticket', {
             type: 'DateTimeUtc',
             access: writeOnlyServerSideFieldAccess,
         },
+        lastCommentWithOrganizationTypeAt: {
+            schemaDoc: 'Time of the last comment with organization type (only from staff user) in ticket',
+            type: 'DateTimeUtc',
+            access: writeOnlyServerSideFieldAccess,
+        },
+        lastCommentWithResidentTypeCreatedByUserType: {
+            schemaDoc: 'User type of the author who created the last comment of type `resident` in the ticket',
```

Comment:
Type of the User who created last comment with type=RESIDENT

Reply:
Please add an example to both of these fields. Their logic is really not obvious